pyproject.toml

```
@@ -0,0 +1,27 @@
+# [build-system]
+# requires = ["hatchling"]
+# build-backend = "hatchling.build"
+
+[project]
+name = "path_generation"
+dynamic = ["version"]
+dependencies = [
+    "numpy"
+    ]
+requires-python = ">=3.8"
+authors = [
+    {name ="Kenneth Chan", email ="kenc0728@gmail.com"},
+    {name = "Pikachu Chan", email = "pikachu!@gmail.com"},
+]
+description = "Example"
+# license = "GPL-3.0-or-later"
+
+[project.scripts]
+path_generation-cli = "path_generation:main_cli"
```

```suggestion
path-generation-cli = "package.__main__:main"
```
IMO, CLI programs should have dashes in their names vs underscores as it's less typing on a standard keyboard. Also, you need to set a proper path to the function which your build system should wrap.